docs/sources/panels-visualizations/visualizations/geomap/index.md

```diff
@@ -282,6 +282,16 @@ The GeoJSON layer allows you to select and load a static GeoJSON file from the f
 - **Add style rule** creates additional style rules.
 - **Display tooltip** allows you to toggle tooltips for the layer.
 
+{{% admonition type="note" %}}
+Styles can be set within the "properties" object of the GeoJSON with support for the following geometrys.
```

Comment:
```suggestion
Styles can be set within the `properties` object of the GeoJSON with support for the following geometries:
```

Reply:
Good suggestions, but will keep double quotes which matches the format of the json standard.